packages/components/organisms/f-offers/src/store/offers.module.js

```diff
@@ -0,0 +1,123 @@
+import axios from 'axios';
+import jwtDecode from 'jwt-decode';
+import {
+    MUTATION_SET_AUTH_TOKEN,
+    MUTATION_SET_BRAZE_KEY,
+    MUTATION_SET_CUSTOMER_DATA,
+    ACTION_INITIALISE_OFFERS,
+    ACTION_HANDLE_CUSTOMER_DATA
+} from './types';
+
+/**
+ * Setup the config for axios
+ * @param authHeader
+ * @param timeout
+ * @returns {{headers: {Authorization, "Content-Type": string}, timeout}}
+ */
+const createAxiosConfig = (authHeader, timeout) => ({
+    headers: {
+        'Content-Type': 'application/json',
+        Authorization: authHeader
+    },
+    timeout
+});
+
+/**
+ * @function resolveCustomerDetails
+ * @param  {object} data  - Api response object.
+ * @param  {string} authToken - auth token.
+ */
+const resolveCustomerDetails = (data, authToken) => {
+    const tokenData = jwtDecode(authToken);
+    return {
+        phoneNumber: data?.customer?.phoneNumber ? data?.customer?.phoneNumber :
+            (tokenData?.phone_number || tokenData?.mobile_number),
```

Comment:
The place I linked before checks mobile_data then falls back to phone_number, could we reverse the order of these to be consistent, unless there's a reason it needs to be different here?
```suggestion
            (tokenData?.mobile_number || tokenData?.phone_number),
```

Reply:
yep good point will address in a follow up PR